WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -253,6 +253,47 @@ public object[] GetHistory()
 			}).ToArray();
 	}
 
+	[JsonRpcMethod("excludefromcoinjoin")]
+	public IEnumerable<OutPoint> ExcludeFromCoinjoin(bool exclude, string? password = null, OutPoint[]? outPoints = null, string[]? labels = null)
+	{
+		var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+
+		AssertWalletIsLoaded();
+		AssertWalletIsLoggedIn(activeWallet, password ?? "");
+
+		var updatedCoinsOutPoints = new List<OutPoint>();
+		foreach (var coin in activeWallet.GetAllCoins())
```

Comment:
RPC methods are just another interface to business logic. There should not be logic here (loops, control flow statements) except to build the outputs.

Reply:
I moved it to Wallet